src/App.js

```diff
@@ -1,9 +1,53 @@
+/* eslint-disable import/extensions */
+/*
+reason for this is that I did try to change it several times
+and it gave me contradicting rules
+namely:
+- Missing file extension for "./components/Calculator"  import/extensions
+- Unexpected use of file extension "js" for "./components/home.js"   import/extensions
+- Missing file extension for "./components/home"
+*/
+import {
+  BrowserRouter, Routes, Route, Outlet,
+} from 'react-router-dom';
 import './App.css';
 import Calculator from './components/calculator';
+import Home from './components/home';
+import Quote from './components/quote';
+
+/* eslint-disable import/extensions */
+
+function Layout() {
+  return (
+    <>
+      <nav>
+        <h1 className="heading">Math Geeks</h1>
+        <ul>
+          <li><a href="/">Home</a></li>
+          <li><a href="calculator">Calculator</a></li>
+          <li><a href="quote">Quote</a></li>
+        </ul>
+      </nav>
+      <Outlet />
```

Comment:
- [Optional] Please it would be better to use `NavLink` from react-router-dom since Navlink Prevent default behavior: When you click on a standard <a> tag, it will typically trigger a full page refresh. However, with NavLink, you can prevent this default behavior by using the onClick prop and calling the preventDefault() method. This can be useful if you want to handle navigation within your application without triggering a full page refresh.